### DISCUSSION TITLE
installation tutorial

### DISCUSSION THREAD

**Discussion by 1Kaux** [gennariarmando/v-hud#82] [Q&A] [answered]
Please make an installation tutorial :C

**Comment by gennariarmando** [comment, marked as answer]
https://github.com/gennariarmando/v-hud#installation

**Reply by 1Kaux** [reply]
I ask for the tutorial because I don't know how to understand it. I don't understand the Ultimate ASI Loader thing, I searched and searched for it and I didn't find anything. I already tried several ways to install and nothing

**Comment by 1Kaux** [comment]
I ask for the tutorial because I don't know how to understand it. I don't understand the Ultimate ASI Loader thing, I searched and searched for it and I didn't find anything. I already tried several ways to install and nothing
